### PR TITLE
Edit profile icons

### PR DESCRIPTION
**Pull request by edyhtan**
-Added edit profile page that support icon upload
-Added backend route for editing and accessing user information
-Changed existing profile images to load from the backend.

### CODE REVIEW

**Comment by edyhtan** [comment]
image

Edit Profile Page UI.
Note that both edit personal info and reset password is currently disabled. 

The Icon will not update before the user clicked the save button.

File: client/src/components/Navbar.js [hunk 1]
```diff
@@ -53,6 +54,16 @@ const useStyles = makeStyles((theme) => ({
...
+    const [iconURL, setIconURL] = useState(process.env.PUBLIC_URL + 'images/avatar-1.png')
+
+    useEffect(() => {
+        
+        getProfile((data) => {
+            setIconURL(data.icon)
+        }, (error) => {
+            console.log(error)
+        })
+    })
```
Comment (bonnieli) [inline client/src/components/Navbar.js:66]: i think you need a dependency here for the useEffect

File: client/src/pages/EditProfilePage.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    top: 5%;
+}
+
+.float-div {
+    width: 70%;
+    float: left;
+    padding: 20px;
+}
+
+.avatar {
```
Comment (bonnieli) [inline client/src/pages/EditProfilePage.css]: let's stay consistent on how we style things. 

In another component (https://github.com/hatchways/team-ninja-turtles/blob/dev/client/src/components/ContestCard.js) for example, we use makeStyles

**Review by bonnieli — APPROVED** [review]

File: client/src/components/Navbar.js (hunk 1 above)
Comment (edyhtan) [reply client/src/components/Navbar.js:66]: This will be replaced once we use UserContext.

**Pull request merged**